libraries/objectDefaultFiles/object.js

```diff
@@ -715,6 +728,18 @@
                 this.wasToolJustCreated = makeSendStub('wasToolJustCreated');
                 this.setPinned = makeSendStub('setPinned');
                 this.promptForArea = makeSendStub('promptForArea');
+
+                this.analyticsAdd = makeSendStub('analyticsAdd');
+                this.analyticsRemove = makeSendStub('analyticsRemove');
+                this.analyticsSetCursorTime = makeSendStub('analyticsSetCursorTime');
+                this.analyticsSetHighlightRegion = makeSendStub('analyticsSetHighlightRegion');
+                this.analyticsSetDisplayRegion = makeSendStub('analyticsSetDisplayRegion');
+                this.analyticsSetLens = makeSendStub('analyticsSetLens');
+                this.analyticsSetLensDetail = makeSendStub('analyticsSetLensDetail');
+                this.analyticsSetSpaghettiAttachPoint = makeSendStub('analyticsSetSpaghettiAttachPoint');
+                this.analyticsSetSpaghettiVisible = makeSendStub('analyticsSetSpaghettiVisible');
+                this.analyticsSetAllClonesVisible = makeSendStub('analyticsSetAllClonesVisible');
```

Comment:
Looking at this makes me want to refactor them into a `this.analytics` object in the spatial interface, though it doesn't really make a difference